src/mpid/ch4/netmod/include/netmod.h

```diff
@@ -467,6 +468,10 @@ typedef int (*MPIDI_NM_mpi_ineighbor_alltoallw_sched_t) (const void *sendbuf,
                                                          const MPI_Aint rdispls[],
                                                          const MPI_Datatype recvtypes[],
                                                          MPIR_Comm * comm, MPIR_Sched_t s);
+typedef void (*MPIDI_NM_algorithm_parser_t) (MPIDU_SELECTION_coll_id_t coll_id, int *cnt_num,
+                                             MPIDIG_coll_algo_generic_container_t * cnt,
+                                             char *value);
+typedef const void *(*MPIDI_NM_get_default_container_t) (MPIDU_SELECTION_coll_id_t coll_id);
```

Comment:
`cnt` is typical short for "count". I would rather use the whole word `container` here. However, even "container" is confusing. "container" makes me think of collection container in C++'s STL. How about call them "coll_algo_fn"?

Reply:
`cnt` has been used for containers long before this PR. If you want to update the terminology, it has to be updated everywhere and that's not in the scope of this PR. I think it has to be a separate  PR.

Reply:
As per our discussion, `cnt` will be changed to `coll_t` everywhere in a separate PR.